crates/namada/src/ledger/native_vp/multitoken.rs

```diff
@@ -145,6 +145,20 @@ where
         all_tokens.extend(dec_mints.keys().cloned());
 
         all_tokens.iter().try_for_each(|token| {
+            if token.is_internal()
```

Comment:
this replaces the special token addresses "redirect" from `crates/state/src/write_log.rs`